src/app/zap-templates/zcl/data-model/chip/scene.xml

```diff
@@ -204,17 +204,17 @@ limitations under the License.
         The command is generated in response to a received unicast EnhancedAddScene command
       </description>
       <arg name="Status" type="ENUM8"/>
-      <arg name="GroupId" type="group_id"/>
-      <arg name="SceneId" type="INT8U"/>
+      <arg name="GroupID" type="group_id"/>
+      <arg name="SceneID" type="INT8U"/>
     </command>
 
     <command source="server" code="0x41" name="EnhancedViewSceneResponse" optional="true" disableDefaultResponse="true">
       <description>
         The command is generated in response to a received unicast EnhancedViewScene command
       </description>
       <arg name="Status" type="ENUM8"/>
-      <arg name="GroupId" type="group_Id"/>
-      <arg name="SceneId" type="INT8U"/>
+      <arg name="GroupID" type="group_Id"/>
```

Comment:
some types are `group_id` and some are `group_Id`. Should we normalize casing on types as well?

Reply:
We probably should, as a followup; right now ZAP treats type names as kinda-case-insensitive-except-if-it's-a-named-type or something.  So changing the casing there does not affect codegen and we can do it any time easily.